browser/ui/tabs/split_view_browser_data.h

```diff
@@ -90,14 +90,14 @@ class SplitViewBrowserData : public BrowserUserData<SplitViewBrowserData> {
 
  private:
   friend BrowserUserData;
-  friend class SplitViewBrowserDataUnitTest;
-  friend class SplitViewTabStripModelAdapterUnitTest;
+  friend class SplitViewBrowserDataBrowserTest;
```

Comment:
shouldn't we remove unittests sources if we converted them?

Reply:
Yes, thank you! Fixed here https://github.com/brave/brave-core/pull/25340/commits/7cee20f96eeeaf7b4b71fbd4e87e30f5f5c1335f